documentation.html

```diff
@@ -8,6 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/png" href="img/icon.ico">
     <link rel="stylesheet" href="site.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
```

Comment:
I am not a fan of relying on 3rd part CDNs

Reply:
Oh! Well, I am going to use html css only then. Thanks @marclaporte for review

Reply:
To be clear, I am a fan of Bootstrap, so you can add that. Just that the repo should be self-contained, and you should be able to work on it offline.

Reply:
I put this